docs/building-spark.md

```diff
@@ -130,9 +130,7 @@ To produce a Spark package compiled with Scala 2.11, use the `-Dscala-2.11` prop
     dev/change-version-to-2.11.sh
     mvn -Pyarn -Phadoop-2.4 -Dscala-2.11 -DskipTests clean package
 
-Scala 2.11 support in Spark does not support a few features due to dependencies
-which are themselves not Scala 2.11 ready. Specifically, Spark's external 
-Kafka library and JDBC component are not yet supported in Scala 2.11 builds.
+Scala 2.11 support in Spark does not yet support Spark's external JDBC component.
```

Comment:
Sorry to nitpick, but "external" doesn't make sense here (it was a modifier to Kafka support which is in our /external directory). It makes most sense to say that the JDBC/ODBC server is not supported.